ImprovedAnvil/resources/animations/ini/anvil_animated_plate_bronze.ini

```diff
@@ -0,0 +1,59 @@
+// MANI anvil_animated_plate_bronze (uses transluncency against vanilla)
+
+[general]
+animation_type=e000
+move_scale=7
+ellipse=24
+color_blood=27
+color_chunks=255
+sound_freq=10
+personal_space=3
+cast_frame=4
+walk_sound=WAL_MT
+
+[monster_icewind]
+detected_by_infravision=0
+resref=MANI
+translucent=1
```

Comment:
This is the tag copied from Ghost's own INIi animation file which achieves the transparency.